plugins/cisco_asa/icon_cisco_asa/util/api.py

```diff
@@ -74,6 +74,36 @@ def create_address_object(self, name: str, object_type: str, address: str) -> di
             json_data={"name": name, "host": {"kind": object_type, "value": address}},
         )
 
+    def cli(self, commands: list) -> dict:
+        return self._call_api("POST", "cli", json_data={"commands": commands})
+
+    def get_blocked_hosts(self) -> list:
+        response = self.cli(["show shun"]).get("response", [])
+        blocked_hosts = []
+        if response and isinstance(response, list):
```

Comment:
```suggestion
        if response and isinstance(response, list) and len(response) > 0:
```

Reply:
This additional condition is not necessary. The `if response` expression handles cases where `response` is returned as an empty list.